hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -411,6 +411,12 @@
             Verify pinmux can select the life_cycle, RISC-V, and DFT taps after reset.
             Verify that in TEST_UNLOCKED* and RMA states, pinmux can switch between the three TAPs
             without issuing reset.
+            Verify in PROD state, only the LC tap can be selected.
+            Verify in PROD and DEV states, only the LC tap and RISC-V taps can be selected.
```

Comment:
I think this should say `Verify in DEV state...`

Reply:
oh, that makes more sense. Thanks for correcting me